reg_tests/chgres_cube/driver.hera.sh_Jeff

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
```

Comment:
This version should be removed.

Reply:
@GeorgeGayno-NOAA Just removed that script.